sdk/cxx/include/sawtooth/transaction_handler.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2017 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+------------------------------------------------------------------------------*/
+#pragma once
+#include <list>
+#include <string>
+#include <vector>
+
+#include "proto/transaction.pb.h"
+#include "sawtooth/global_state.h"
+
+namespace sawtooth {
+
+class TransactionHandler {
+ public:
+    virtual std::string transaction_family_name() = 0;
```

Comment:
Should these return `const &` instead of copies???

Should they be marked as `const` methods?